aspnetcore/whats-new/dotnet-AspNetCore.Docs-mod2.md

```diff
@@ -1,68 +1,222 @@
 ---
-title: "ASP.NET Core docs: What's new for August 2023"
-description: "What's new in the ASP.NET Core docs for August 2023."
-ms.custom: August-2023
-ms.date: 09/01/2023
+title: "ASP.NET Core docs: What's new for February 2024"
+description: "What's new in the ASP.NET Core docs for February 2024."
+ms.custom: February-2024
+ms.date: 03/01/2024
 ---
 
-# ASP.NET Core docs: What's new for August 2023
+# ASP.NET Core docs: What's new for February 2024
 
-Welcome to what's new in the ASP.NET Core docs for August 2023. This article lists some of the major changes to docs during this period.
+Welcome to what's new in the ASP.NET Core docs for February 2024. This article lists some of the major changes to docs during this period.
+
+## Miscellaneous
+
+### Updated articles
+
+- [Title not found in: #31913](index.yml) - Hub page update per data and OKR
+- [Title not found in: #31555](zone-pivot-groups.yml) - Blazor Web App with OIDC article
```

Comment:
```suggestion
```

Reply:
@guardrex  @tdykstra this is weird

Reply:
I did have a change to the zone pivot groups file, so it picked that up.